px-vis-xy-chart.html

```diff
@@ -851,6 +892,39 @@
 
       }, 5);
     },
+    _calcMarkerMargins: function() {
+      if(this.hasUndefinedArguments(arguments)) {
+        return;
+      }
+
+
+      var newMarg;
```

Comment:
No need to separate the declaration from the initialization, remove this and add `var` to line 905.

Reply:
Done: Updated changes in new commit.